.github/workflows/release.yml

```diff
@@ -1,15 +1,17 @@
-name: Tag release
+# Use this workflow to trigger stable releases, both minor releases and patches
 
-run-name: "Tag release '${{ inputs.git-ref || github.ref_name }}' (publish: ${{ inputs.publish || github.event_name == 'push' }}, latest: ${{ inputs.latest || github.event_name == 'schedule' }})"
+name: Stable release
+
+run-name: "Stable release '${{ inputs.branch }}' (publish: ${{ inputs.publish }}, latest: ${{ inputs.branch == 'releases/beta' || inputs.latest }})"
 
 on:
   workflow_dispatch:
     inputs:
-      git-ref:
+      branch:
         required: true
         type: string
-        description: "The github ref of this release. If you are publishing it, use a tag (i.e. v1.0.0)."
-        default: main
+        description: "The github branch of this release. Should be 'releases/beta' or 'patches/x.y'"
+        default: releases/beta
       latest:
```

Comment:
do we need this input? It seems we only want `releases/beta` to be latest.

I added it originally in case we wanted to publish an arbitrary git sha and treat it as latest.

Reply:
Yes we need it because we will not only be releasing from `releases/beta`. That's only for promoting beta to stable. We will also need to release security patches from `patches/x.y` using the same workflow.